tools/build/build_macos_arm64.sh

```diff
@@ -9,7 +9,7 @@
 
 set -euo pipefail
 
-python_versions="3.8.19 3.9.19 3.10.14 3.11.9 3.12.5 3.13.0"
+python_versions="3.9.19 3.10.14 3.11.9 3.12.5 3.13.0"
```

Comment:
This file to be deleted in another PR because its deletion has little to do with dropping Python 3.8.